style.css

```diff
@@ -93,4 +101,212 @@ body {
+  .project-details {
+    flex-basis: 29.5rem;
+  }
+
+  .project-details ul {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    gap: 0 1.2rem;
+    margin: 0;
+    width: 85%;
+    justify-content: center;
+  }
+
+  .project-details ul li {
+    text-align: center;
+  }
+
+  .socialmedia ul li {
+    flex: 1;
+  }
+
+  .project-details ul li:first-child {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #344563;
+    font-weight: 600;
+    flex: 1;
+  }
+
+  .project-details ul li:nth-child(3) {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #7a869a;
+    font-weight: 600;
+    flex: 3;
+  }
+
+  .project-details ul li:nth-child(5) {
+    font-size: 1.3rem;
+    font-style: normal;
+    line-height: 1.6rem;
+    color: #7a869a;
+    font-weight: 600;
+    flex: 1;
+  }
+
+  .project p {
+    flex-basis: 29.5rem;
+    margin: 0;
+  }
+
+  .project-tech-stack {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    flex-basis: 29.5rem;
+    display: flex;
+    gap: 0 0.8rem;
+  }
+
+  .project-header img {
+    width: 29.5rem;
+    height: 22rem;
+  }
+
+  .project-tech-stack li a {
+    font-style: normal;
+    font-size: 1.2rem;
+    line-height: 1.6rem;
+    color: #6070ff;
+    background-color: #ebebff;
+    padding: 0.4rem 1.2rem;
+    border-radius: 0.8rem;
+  }
+
+  .project-tech-stack a:hover {
+    text-decoration: underline;
+  }
+
+  .btn {
+    width: auto;
+    display: flex;
+    align-items: center;
+    font-size: 1.7rem;
+    line-height: 2.4rem;
+    font-weight: 500;
+    color: #396df2;
+    border: 0.1rem solid #396df2;
+    border-radius: 0.8rem;
+    padding: 1.2rem;
+  }
+
+  .btn:hover {
+    background-color: #6070ff;
+    color: #fff;
+    box-shadow: 5px 5px 15px 5px rgba(64, 83, 252, 0.24);
+  }
+
+  .btn:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .about-myself {
+    background-color: #fff;
+    height: auto;
+    display: flex;
+    flex-wrap: wrap;
+    align-content: center;
+    gap: 1.2rem 0;
+    margin: 11.4rem 0 0 0;
+    padding: 0 2.4rem;
+    border-top-right-radius: 15%;
+  }
+
+  .about-myself .title {
+    font-weight: bold;
+    font-size: 4rem;
+    line-height: 5.2rem;
+    color: #172b4d;
+    flex-basis: 32.7rem;
+    margin-top: 11.4rem;
+  }
+
+  .about-myself p {
+    color: #344563;
+    font-size: 1.6rem;
+    font-weight: normal;
+    line-height: 2.4rem;
+    flex-basis: 32.7rem;
+    margin: 0;
+  }
+
+  .contactme {
+    color: #7f8cff;
+    font-size: 1.6rem;
+    font-weight: 400;
+    line-height: 2.4rem;
+    flex-basis: 100%;
+  }
+
+  .socialmedia {
+    flex-basis: 100%;
+  }
+
+  .socialmedia ul {
+    list-style: none;
+    display: flex;
+    width: 20rem;
+  }
+
+  .about-myself .skills {
+    flex-basis: 100%;
+  }
+
+  .about-myself .skills ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    margin: 0;
+  }
```

Comment:
- [ ] Please make sure to stick to the `Figma` design template as much as possible. Currently, Your skills section is missing some parts of the design shown on the `Figma` template so kindly add it. I have highlighted the missing parts in blue in the below illustrations. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="374" alt="Screenshot 2023-06-02 at 12 08 21" src="https://github.com/AjroudiRida/Portfolio/assets/107192076/0688fe14-c86e-4356-bf28-cde07e503472">
</td>
<td>
<img width="344" alt="Screenshot 2023-06-02 at 12 09 12" src="https://github.com/AjroudiRida/Portfolio/assets/107192076/99a00e69-9ad5-42a1-929a-362746a81152">
</td>
</tr>
</table>